recipes/openroad/meta.yaml

```diff
@@ -0,0 +1,91 @@
+{% set name = "openroad" %}
+{% set version = "2023.12.14" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/Precision-Innovations/OpenROAD/archive/refs/tags/{{ version | replace(".", "-") }}.tar.gz
+    sha256: f364a5267a6cae1b801382f7abb4ad4b76dbd02e01a3131f51ead9a71d0795af
+
+  # TODO(proppy): honor conda forge `tarballs-no-repos` policy:
```

Comment:
@conda-forge/help-c-cpp, any guidance on the vendoring of `OpenSTA` and `abc` dependencies? Those are actually fork that are maintained by the same upstream (and included in the original project as a submodule), is it fine to include them as `git_url` in the recipe?